test/e2e/pages/vl-form-validation.page.js

```diff
@@ -1,14 +1,107 @@
 const VlFormValidation = require('../components/vl-form-validation');
-const { Page, Config } = require('vl-ui-core').Test;
+
+const { Page, Config, VlElement } = require('vl-ui-core').Test;
+const { By } = require('vl-ui-core').Test.Setup;
+const { Key } = require('selenium-webdriver');
+
 
 class VlFormValidationPage extends Page {
-    async _getFormValidation(selector) {
-        return new VlFormValidation(this.driver, selector);
+
+    async getFormMetVerplichtVeld() {
+        return new DummyForm(this.driver, '#form-met-verplicht-veld');
+    }
+
+    async getFormMetVerplichtEmailVeld() {
+        return new DummyForm(this.driver, '#form-met-email-veld');
+    }
+
+    async getFormMetVerplichtIbanVeld() {
+        return new DummyForm(this.driver, '#form-met-iban-veld');
+    }
+
+    async getFormMetVerplichtTelefoonnummerVeld() {
+        return new DummyForm(this.driver, '#form-met-telefoonnr-veld');
+    }
+
+    async getFormMetVerplichtDatumVeld() {
+        return new DummyForm(this.driver, '#form-met-datum-veld');
+    }
+
+    async getFormMetVerplichtRRNVeld() {
+        return new DummyForm(this.driver, '#form-met-rrn-veld');
+    }
+
+    async getFormMetVerplichtSelectVeld() {
+        return new DummyForm(this.driver, '#form-met-select-veld');
+    }
+
+    async getSuccessFormMetVerplichtVeld() {
+        return new DummyForm(this.driver, '#form-success-met-verplicht-veld');
+    }
+
+    async getFormZonderValidatie() {
+        return new DummyForm(this.driver, '#form-zonder-validatie');
     }
 
     async load() {
-        await super.load(Config.baseUrl + '/demo/vl-form-validation.html');
+        await super.load(Config.baseUrl + '/demo/vl-form-validation-e2e.html');
+    }
+}
+
+class DummyForm extends VlElement {
```

Comment:
Deze klassen lijken eerder een "component object" dan een "page object" te zijn.
Is hier niks generieks voor?

Reply:
We wilden onze eigen 'vl-elementen' niet gebruiken omdat we anders (circulaire) dependencies krijgen. Daarom hebben we dummy componenten gemaakt om vl-form-validation te kunnen e2e testen.

Reply:
Spijtig dat we dan eigenlijk voor een stuk duplicatie lijken te krijgen?
Ik zou ze alleszins sowieso onder de components folder steken ipv. de pages folder (want het zijn components). En daar dan niet in dezelfde file als van de form-validation zelf.

Reply:
ok, ik verhuis deze...